From 346ad00dca160ac8402d221664454fa1d5b96c03 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Thu, 27 Oct 2022 10:33:34 -0700
Subject: [PATCH 02/13] Do not build kojivm

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 1c21385..da8fa31 100644
--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,6 @@
 NAME=koji
 SPECFILE = $(firstword $(wildcard *.spec))
-SUBDIRS = hub builder koji cli util www plugins vm
+SUBDIRS = hub builder koji cli util www plugins
 
 ifndef PYTHON
 export PYTHON=python2
-- 
2.37.3

