From f2577cdfb6f262c98fefe497e60f05326014dc91 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Thu, 20 Jun 2024 11:10:46 -0700
Subject: [PATCH 10/10] Remove version pin for python-mock

Signed-off-by: William Douglas <william.douglas@intel.com>
---
 test-requirements.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test-requirements.txt b/test-requirements.txt
index 897624f..6bcb481 100644
--- a/test-requirements.txt
+++ b/test-requirements.txt
@@ -2,7 +2,7 @@
 defusedxml
 flake8
 flake8-import-order
-mock<=2.0.0
+mock
 requests-mock
 coverage
 pytest
-- 
2.45.1

